docs/provisioning_guide/src/asciidoc/_chapters/prepare.adoc

```diff
@@ -0,0 +1,379 @@
+distribution manager's REST API.
+| REST_BUILD         | Tar file containing the REST component.                        | None | Not needed if using a Trafodion package installation tar file.
+| SQ_ROOT            | Target directory for the Trafodion software.                   | $HOME_DIR/trafodion           | Trafodion is installed in this directory on all nodes in `$NODE_LIST`.
+| START              | Whether to start Trafodion after install/upgrade.              | N                             | Does not apply to Recipe-Based Provisioning.
+| SUSE_LINUX         | Whether your installing Trafodion on SUSE Linux.               | false                         | Auto-detected by the Trafodion Installer.
+| TRAF_PACKAGE       | The location of the Trafodion installation package tar file or core installation tar file. | None | The package file contains the Trafodion server,
+DCS, and REST software while the core installation file contains the Trafodion server software only. If you're using a core installation file, then you need to
+record the location of the DCS and REST installation tar files, too. Normally, you perform Trafodion provisioning using a Trafodion package installation tar file.
+| TRAF_USER          | The Trafodion runtime user ID.                                  | trafodion                     | Must be `trafodion` in this release.
+| TRAF_USER_PASSWORD | The password used for the `trafodion:trafodion` user ID.       | traf123                       | Must be 6-8 characters long.
+| URL                | FQDN and port for the Distribution Manager's REST API.         | None                          | Include `http://` or `https://` as applicable. Specify in the form:
+`<IP-address>:<port>` or `<node name>:<port>` Example: `https://susevm-1.yourcompany.local:8080`
+|===
+
+1. The ID matches the environmental variables used in the Trafodion Installation configuration file. Refer to <<install-trafodion-installer,Trafodion Installer>>
+for more information.
+2. Refer to <<enable-security,Enable Security>> for more information about these security settings.
+
+
+<<<
+[[prepare-install-required-software-packages]]
+== Install Required Software Packages
+
+[[prepare-download-and-install-packages]]
+=== Download and Install Packages
+
+The step is required if you're:
+
+* Installing Trafodion on SUSE.
+* Using Recipe-Based Provisioning.
+* Can't download the required software packages using the Internet.
+
+If none of these situations exist, then we highly recommend that you use the Trafodion Installer.
+
+You perform this step as a user with `root` or `sudo` access.
+
+Install the packages listed in <<requirements-software-packages,Software Packages>> above on all nodes in the cluster. Note the special
+handling for `log4c&#43;&#43;`. See <<prepare-build-and-install-log4cplusplus, Build and Install log4c++>> below for more information.
```

Comment:
Yep. Markup-language limitations.

On Wed, Feb 17, 2016 at 1:17 PM, DaveBirdsall <notifications@github.com>
wrote:

> In docs/provisioning_guide/src/asciidoc/_chapters/prepare.adoc
> <https://github.com/apache/incubator-trafodion/pull/326#discussion_r53224885>
> :
>
> > +
> > +[[prepare-download-and-install-packages]]
> > +=== Download and Install Packages
> > +
> > +The step is required if you're:
> > +
> > +* Installing Trafodion on SUSE.
> > +* Using Recipe-Based Provisioning.
> > +* Can't download the required software packages using the Internet.
> > +
> > +If none of these situations exist, then we highly recommend that you use the Trafodion Installer.
> > +
> > +You perform this step as a user with `root` or `sudo` access.
> > +
> > +Install the packages listed in <<requirements-software-packages,Software Packages>> above on all nodes in the cluster. Note the special
> > +handling for `log4c&#43;&#43;`. See <<prepare-build-and-install-log4cplusplus, Build and Install log4c++>> below for more information.
>
> Three different ways to render "++"! :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/326/files#r53224885>.
>



-- 
Thanks,

Gunnar
*If you think you can you can, if you think you can't you're right.*